--- Makefile	2008-07-29 00:33:38.000000000 +0200
+++ Makefile.oden	2008-12-21 22:45:34.000000000 +0100
@@ -102,7 +102,6 @@
 install: all
 	test -d $(DESTDIR)$(BINDIR) || mkdir -p $(DESTDIR)$(BINDIR)
 	$(UCBINSTALL) -c mailx $(DESTDIR)$(BINDIR)/mailx
-	$(STRIP) $(DESTDIR)$(BINDIR)/mailx
 	test -d $(DESTDIR)$(MANDIR)/man1 || mkdir -p $(DESTDIR)$(MANDIR)/man1
 	$(UCBINSTALL) -c -m 644 mailx.1 $(DESTDIR)$(MANDIR)/man1/mailx.1
 	test -d $(DESTDIR)$(SYSCONFDIR) || mkdir -p $(DESTDIR)$(SYSCONFDIR)
