diff -Nru mailx-12.4.orig/openssl.c mailx-12.4/openssl.c
--- mailx-12.4.orig/openssl.c	2007-08-04 13:38:03.000000000 +0200
+++ mailx-12.4/openssl.c	2016-03-03 05:15:57.749891140 +0100
@@ -211,9 +211,11 @@
 
 	cp = ssl_method_string(uhp);
 	if (cp != NULL) {
+#ifndef OPENSSL_NO_SSL2
 		if (equal(cp, "ssl2"))
 			method = SSLv2_client_method();
-		else if (equal(cp, "ssl3"))
+#endif
+		if (equal(cp, "ssl3"))
 			method = SSLv3_client_method();
 		else if (equal(cp, "tls1"))
 			method = TLSv1_client_method();
 
